src/components/Job/jobTicket.test.tsx

```diff
@@ -1,15 +1,39 @@
-import React from 'react';
-import { render } from '@testing-library/react';
-import JobTicket from './JobTicket';
-describe('JobTicket', () => {
-  it('renders correctly', () => {
-    const { getByText } = render(<JobTicket />);
-    expect(getByText('Mid/Senior Fullstack .Net Developer (Angular, ReactJS)')).toBeInTheDocument();
-    expect(getByText('$')).toBeInTheDocument();
-    expect(getByText('Sign in to view')).toBeInTheDocument();
-    expect(getByText('c++')).toBeInTheDocument();
-    expect(getByText('html')).toBeInTheDocument();
-    expect(getByText('Ho Chi Minh')).toBeInTheDocument();
-    expect(getByText('3h')).toBeInTheDocument();
+import React from "react";
+import { render } from "@testing-library/react";
+import JobTicket from "./JobTicket";
+import { JobModal } from "../../type/type";
+
+describe("JobTicket component", () => {
+  const item:JobModal = {
+    createdAt: "2023-03-15T22:08:58.057Z",
+    title: "Global Factors Developer",
+    avatar: "https://loremflickr.com/640/480/business",
+    address: "Cerritos",
+    tag: "Strategist",
+    id: "1"
+  };
```

Comment:
using the variable item in case test.
exp getByText(item.id);

Reply:
i've changed